src/presentation/settings/explorer.tsx

```diff
@@ -86,26 +93,25 @@ const SettingsExplorer: React.FC = () => {
     >
       <p className="font-regular my-8 text-base text-left">Select the explorer</p>
       <Select
-        onClick={() => setCustom(false)}
         list={explorerTypesForNetwork(network)}
         selected={
-          app.explorerByNetwork[network]
-            ? app.explorerByNetwork[network].type
+          currentExplorerURLs
+            ? currentExplorerURLs.type
             : appInitState.explorerByNetwork[network].type
         }
         onSelect={onSelect}
         disabled={false}
       />
 
-      {custom && (
-        <SettingsCustomExplorerForm
-          onDone={() => setCustom(false)}
-          dispatch={dispatch}
-          network={network}
-        />
-      )}
+      <div className="flex justify-end mt-1">
+        <div>
+          <Button type="button" onClick={() => pushToCustomForm()}>
+            Use Custom
```

Comment:
This should be update

Reply:
https://www.loom.com/share/00dc8928f3274697980aed90e4aea283?from_recorder=1&focus_title=1